pkgs/by-name/eb/eb-garamond/package.nix

```
@@ -1,23 +1,40 @@
 {
   lib,
   stdenvNoCC,
-  fetchzip,
+  fetchFromGitHub,
+  python3,
+  ttfautohint,
 }:
-
 stdenvNoCC.mkDerivation rec {
   pname = "eb-garamond";
   version = "0.016";
 
-  src = fetchzip {
-    url = "https://bitbucket.org/georgd/eb-garamond/downloads/EBGaramond-${version}.zip";
-    hash = "sha256-P2VCLcqcMBBoTDJyRLP9vlHI+jE0EqPjPziN2MJbgEg=";
+  src = fetchFromGitHub {
+    owner = "georgd";
+    repo = "EB-Garamond";
+    tag = "v${version}";
+    hash = "sha256-ajieKhTeH6yv2qiE2xqnHFoMS65//4ZKiccAlC2PXGQ=";
   };
 
+  nativeBuildInputs = [
+    (python3.withPackages (p: [ p.fontforge ]))
+    ttfautohint
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make TAG_COMMIT=${src.rev} TAG=v${version} COMMIT=${src.rev} DATE=19700101
```

```suggestion
    make
```

While these variables have been introduced in the [master](https://github.com/georgd/EB-Garamond/blob/master/Makefile) branch, they do not exist in [0.016](https://github.com/georgd/EB-Garamond/blob/v0.016/Makefile)